spec/graphql/relay/page_info_spec.rb

```diff
@@ -75,8 +75,8 @@ def get_last_cursor(result)
       result = star_wars_query(query_string, "last" => 1, "first" => 1, "before" => cursor_of_last_base)
       assert_equal(true, get_page_info(result)["hasNextPage"])
       assert_equal(true, get_page_info(result)["hasPreviousPage"])
-      assert_equal("Mg==", get_page_info(result)["startCursor"])
-      assert_equal("Mg==", get_page_info(result)["endCursor"])
+      assert_equal("MQ==", get_page_info(result)["startCursor"])
+      assert_equal("MQ==", get_page_info(result)["endCursor"])
```

Comment:
It looks like this cursor value changed by 1, is that correct? Can you help me understand the _meaning_ of this change? (What did it _used_ to point to, what does it point to now?)